testnets/nilliontestnet/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "nilliontestnet",
+  "assets": [
+    {
+      "description": "NIL - the native token of Nillion",
+      "denom_units": [
+        {
+          "denom": "unil",
+          "exponent": 0
+        },
+        {
+          "denom": "nil",
+          "exponent": 6
+        }
+      ],
+      "base": "unil",
+      "name": "Nillion",
+      "display": "nil",
+      "symbol": "NIL",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/nilliontestnet/images/nil.png"
```

Comment:
Can I ask why the asset only has the png and the chain only has the svg? It seems like they can be be added to both since they are visually identical

Reply:
Images now contain both svg and png

Reply:
Because the images are visual identical, please put them in the same object.

```
{
  png:
  svg:
}
```